packages/manager/modules/bm-server-components/src/ftp-backup-storage/access/add/dedicated-server-ftp-backup-access-add.controller.js

```diff
@@ -0,0 +1,111 @@
+import {
+  DEDICATED_SERVER_FTP_BACKUP_IP_BLOCKS_LIMIT,
+  FTP_BACKUP_STORAGE_ALERT,
+} from '../../ftp-backup.constants';
+
+export default class AddAccessFtpBackupCtrl {
+  constructor($rootScope, $stateParams, $translate, Alerter, Server) {
+    this.$rootScope = $rootScope;
+    this.$stateParams = $stateParams;
+    this.$translate = $translate;
+    this.Alerter = Alerter;
+    this.Server = Server;
+    this.ipBlocksLimit = DEDICATED_SERVER_FTP_BACKUP_IP_BLOCKS_LIMIT;
+  }
+
+  $onInit() {
+    this.existingIpBlocks = this.results;
+    this.access = {
+      listIp: [],
+      ip: null,
+      ftp: false,
+      cifs: false,
+      nfs: false,
+    };
+    this.loading = false;
+  }
+
+  load() {
+    return () => {
+      this.loading = true;
+      return this.Server.getAuthorizableBlocks(this.$stateParams.productId)
+        .then((list) => {
+          this.access.listIp = list;
+        })
+        .catch(({ data }) => {
+          this.goBack();
+          this.Alerter.alertFromSWS(
+            this.$translate.instant(
+              'server_configuration_ftpbackup_access_add_ip_failure',
+            ),
+            data,
+            FTP_BACKUP_STORAGE_ALERT,
+          );
```

Comment:
I don't think anything after the goBack has been played. The alert management should be done in transition.